src/test/functional/functional-tests.js

```diff
@@ -1940,53 +1902,117 @@ describe('functional tests', function() {
     }))
     after((done) => client.removeBucket(versionedBucketName, done))
 
-
     step(`presignedPutObject(bucketName, objectName, expires=1000, _incoming_obj,cb)_bucketName:${versionedBucketName} ${versionedPresignObjName} _version 1`, done => {
```

Comment:
Do we need `_incoming_obj` here?
I don't see it used anywhere. Am I missing something?

Reply:
Thank you @ebozduman . I have updated.